elements/rh-alert/rh-alert.css

```diff
@@ -123,15 +127,21 @@ footer ::slotted([slot="actions"]) {
   background-color: transparent;
   color: var(--rh-color-interactive-blue-darker, #0066cc);
   font-size: var(--rh-font-size-body-text-sm, 0.875rem);
+  font-family: var(--_font-family);
 }
 
 /* TODO: separate focus and hover styles */
-footer ::slotted([slot="actions"]:hover),
 footer ::slotted([slot="actions"]:focus) {
   text-decoration: underline;
   color: var(--rh-color-interactive-blue-darkest, #004080);
 }
 
+footer ::slotted([slot="actions"]:hover) {
+  cursor: pointer;
+  text-decoration: underline;
+  color: var(--rh-color-interactive-blue-darkest, #004080);
+}
+
 :host(:not([variant])) #container,
 :host([toast]) #container {
```

Comment:
I believe the token for the box shadow on the toast variant should be --rh-box-shadow-lg. That's the closest to the design specs.

Reply:
Updated this to --rh-box-shadow-lg w/ the token values